interface/src/app/gene/gene-network.tpl.html

```diff
@@ -2,17 +2,28 @@
   <h1 class="page-header">
     Gene Network
   </h1>
-  <autocomplete-search-panel organism="searchCtrl.organism" autocomplete="searchCtrl.autocomplete"
-    ng-if="searchCtrl.autocomplete">
+
+  <m-l-model-selector selected-m-l-model="searchCtrl.selectedMLModel">
```

Comment:
I found this tag a little weird. Do you think **ml-model-selector** and **selected-ml-model** (and **ml-model** later) will be better? (This is trivial, if you prefer "m-l-...", I am fine with it.